src/mmw/js/src/compare/models.js

```diff
@@ -43,6 +43,16 @@ var ChartRowsCollection = Backbone.Collection.extend({
         this.scenarios.forEach(function(scenario) {
             scenario.get('results').on('change', update);
         });
+    },
+
+    getScenarioResults: function(typeKey) {
+        return this.scenarios.map(function(scenario) {
+            var resultKey = coreUtils.getTR55ResultKey(scenario),
+                result = scenario.get('results')
+                .findWhere({ name: typeKey })
```

Comment:
Consider indenting these at least one level further in, so they don't align with the left-hand side declaration of variables.